src/engine/Sim3D.ts

```diff
@@ -73,6 +73,7 @@ export class Sim3D extends EventEmitter {
   private cameraManager: CameraManager;
 
   private isRendering = false;
+  private stopPhysics = false;
```

Comment:
Rename this to something like `physicsActive` (just so we have the same truthiness as isRendering)